sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
     ([#25752](https://github.com/Azure/azure-sdk-for-python/pull/25752))
 - Implement all network statsbeat
     ([#25845](https://github.com/Azure/azure-sdk-for-python/pull/25845))
+- Implement attach statsbeat
+    ([#25956](https://github.com/Azure/azure-sdk-for-python/pull/25956))
```

Comment:
I usually add new changes at the top of the section not the bottom. Guess I'll switch that for this package.